arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -2478,6 +2478,225 @@
+        "operationId": "Sessions_CreateOrUpdate",
+        "x-ms-examples": { 
+          "SessionsCreateOrUpdate": { "$ref": "../examples/SessionsCreateOrUpdate.json" } 
+        },
+        "description": "Creates or updates an integration account session.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationAccountName",
+            "description": "The integration account name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "sessionName",
+            "description": "The integration account session name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "session",
+            "description": "The integration account session.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccountSession"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccountSession"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccountSession"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "IntegrationAccountSessions"
+        ],
+        "operationId": "Sessions_Delete",
+        "x-ms-examples": { 
+          "SessionsDelete": { "$ref": "../examples/SessionsDelete.json" } 
+        },
+        "description": "Deletes an integration account session.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationAccountName",
+            "description": "The integration account name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "sessionName",
+            "description": "The integration account session name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Are you sure that you are not returning the deleted resource in a 200 response? The normal expectation for an RP is to return the resource for 200 (indicating that a delete actually took place) and have a 204 returned if the delete request was well formed, but the resource didn't exist.